src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const tableRows = document.querySelectorAll('tr');
+
+[...tableRows].forEach(row => {
+  const secondCellClone = row.children[1].cloneNode(true);
+
+  row.children[3].before(secondCellClone);
```

Comment:
Should be done in front of last column, not the one before last

your result:
![image](https://user-images.githubusercontent.com/37161315/175500178-e8530573-86c5-45ab-a18c-80e113df496a.png)

should be:
![image](https://github.com/mate-academy/js_task_clone_table_column_DOM/raw/master/src/images/result.png)

Reply:
yep, fixed